resources/lib/WatchlistFlavor/AniList.py

```diff
@@ -257,6 +263,12 @@ def _base_next_up_view(self, res):
         poster = image = res['coverImage']['extraLarge']
         plot = None
 
+        if next_up > episode_count:
```

Comment:
```python
if next_up > episode_count and episode_count > 0:
```
This solves the One Piece issue with Null Total episodes in Anilist

Reply:
Thanks for this @Gujal00, sorry was super busy today. Will update the PR.